usaspending_api/submissions/test_load_submission.py

```diff
@@ -1,47 +1,52 @@
-from django.test import TestCase
 from django.core.management import call_command
 from usaspending_api.accounts.models import AppropriationAccountBalances
 from usaspending_api.awards.models import (
     Award, FinancialAccountsByAwards,
-    FinancialAccountsByAwardsTransactionObligations,
-    FinancialAssistanceAward, Procurement)
+    FinancialAccountsByAwardsTransactionObligations, FinancialAssistanceAward,
+    Procurement)
 from usaspending_api.financial_activities.models import FinancialAccountsByProgramActivityObjectClass
 from usaspending_api.references.models import LegalEntity, Location
 from usaspending_api.submissions.models import SubmissionAttributes
 
 import pytest
 
 
-class LoadSubmissionTest(TestCase):
+@pytest.fixture(scope="module")
+def endpoint_data():
+    call_command('flush', '--noinput')
+    call_command('loaddata', 'endpoint_fixture_db')
```

Comment:
This is outside the current scope of my pytest knowledge, but I wonder if there's a way to define a fixture more globally, so that all of our test files can access it. It seems there are a few tests that have to run through this endpoint loading process, and it would be nice if we only had to do it once. Presumably there's a way to rollback any db modifications that are made during an individual test, so that the initial snapshot remains the same from test to test?

This is all purely speculative 😄